### PR TITLE
483: remove default coordinate for radius filter

### PR DESCRIPTION
**Pull request by trbmcginnis**
- make default for `distance_from_point` equal to an empty array `[]`
- make it so that filtering will not occur until user has created a point/selected a centroid, even if the filter is turned on. This is done by checking if `queryOptions.distance_from_point.length` is true (that the array is populated)
- in `filter-distance-from-point.hbs`, check if `pointGeometry` exists before displaying radius circle and zooming to point
- this PR also installs `ember-test-selectors` for better testing and updates some comments on `show-geography.hbs`

Closes #488

### CODE REVIEW

File: app/templates/show-geography.hbs [hunk 1]
```diff
@@ -13,25 +13,25 @@
     mutateArray=(action 'mutateArray')
     appliedFilters=applied-filters
     as |filters|}}
-  {{/filter-mutators}}
+  {{!filter-mutators}}
 
-  Filter-mutators yields a `filters.section` component which helps group together
+  {{!-- Filter-mutators yields a `filters.section` component which helps group together
```
Comment (julialucyhogan) [inline app/templates/show-geography.hbs]: typo?

File: app/templates/show-geography.hbs [hunk 2]
```diff
@@ -13,27 +13,27 @@
     mutateArray=(action 'mutateArray')
     appliedFilters=applied-filters
     as |filters|}}
-  {{/filter-mutators}}
+  {{!filter-mutators}}
```
Comment (julialucyhogan) [inline app/templates/show-geography.hbs]: should still be `/`, right?

File: app/templates/show-geography.hbs [hunk 3]
```diff
@@ -13,27 +13,27 @@
...
-  Filter-mutators yields a `filters.section` component which helps group together
+  {{! Filter-mutators yields a `filters.section` component which helps group together
   the actual filterable fieldnames. For example, some "filters" may actually be affecting
   multiple fields in the database. `filters.section` makes sure those get turned on when
   mutated.
 
   {{#filters.section
     filterNames=(array 'names' 'of' 'grouped' 'fields') as |section|}}
-  {{/filters.section}}
+  {{!/filters.section}}
```
Comment (julialucyhogan) [inline app/templates/show-geography.hbs]: should still be `/`

File: app/templates/show-geography.hbs [hunk 4]
```diff
@@ -13,27 +13,27 @@
...
 
-  this component is abstract because it has no markup - it only sets up some actions needed for
+  {{! this component is abstract because it has no markup - it only sets up some actions needed for
   mutating groups. It gets a `filter-wrapper` component and the `delegate-mutation` action.
 
-  {{#section.filter-wrapper}}
-    This component invokes all the necessary markup for this to work
+  {{! #section.filter-wrapper}}
```
Comment (julialucyhogan) [inline app/templates/show-geography.hbs]: del `!`

**Comment by julialucyhogan** [comment]
also rebase & squash before u merge :) ! !!!!

File: app/templates/show-geography.hbs (hunk 3 above)
Comment (trbmcginnis) [reply app/templates/show-geography.hbs]: If I don't add the !, it uncomments it

**Comment by trbmcginnis** [comment]
> also rebase & squash before u merge :) ! !!!!

I have been! You're too fast for me

**Pull request merged**